SolarTherm/Materials/Geopolymer_Rahjoo_2022.mo

```diff
@@ -0,0 +1,64 @@
+within SolarTherm.Materials;
+
+package Geopolymer_Rahjoo_2022
```

Comment:
provide a link to the data source, and more words of explanation about what it is.